src/rest/connector/tests/test_nd.py

```diff
@@ -46,11 +46,6 @@ def test_connection(self):
             connection.disconnect()
         self.assertEqual(connection.connected, False)
 
-    def test_post_not_connected(self):
-        connection = Rest(device=self.device, alias='rest', via='rest')
-        with self.assertRaises(Exception):
-            connection.post(api_url='temp', payload={'payload':'something'})
-
```

Comment:
this shouldnt be deleted

Reply:
Hi Daniel,

To me, this function seemed duplicated. The same code can be found in line 73-76 in the original file or 68-71 in the current version.
I can re-add it, but please confirm it's not duplicated code.

Thank you,
Romel

P.S.: Sorry if you received multiple notifications.